### PR TITLE
[ANI-9] Creating batches

### PR DESCRIPTION
**Pull request by AldwinVergara**
**Finished Batch Screen (CRUD). Requesting to merge brach with master.

**New Feature:**
-Added a button to the admin's navigation menu that will link to Batch Screen.
-Added intermediary table for batch_farmers that contains the enrolled farmers assigned in a batch.

**Fixes**
-Fixed errors on the batch_farmers table. Batch screen is able to fetch farmer IDs and incorporate them in a batch.
-Fixed errors on site management layout/fields. Batch screen is also able to fetch region, province, and minuipality IDs and save in the table.
-Fixed errors on foreign keys and connection/relations of table migrations.

**Practices:**
-Practiced git commits for every change.
-Practiced merging branch with master for reference and review of other screens.

### CODE REVIEW

File: app/Http/Livewire/CreateBatch.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Http\Livewire;
+
+use Livewire\Component;
+
+class CreateBatch extends Component
```
Comment (jg-rivera) [inline app/Http/Livewire/CreateBatch.php:7]: Kindly delete this unused livewire component

File: app/Models/Batch/Batches.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+use App\Models\Site\Municity;
+use App\Models\Site\Province;
+use App\Models\Site\Region;
+use App\Models\User;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Orchid\Filters\Filterable;
+
+class Batches extends Model
```
Comment (jg-rivera) [inline app/Models/Batch/Batches.php:13]: Model class name should be in singular form: `Batch`.

File: app/Orchid/Layouts/Batch/AddSiteLayout.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+use App\Models\Site\Municity;
+use App\Models\Site\Province;
+use App\Models\Site\Region;
+use Orchid\Screen\Field;
+use Orchid\Screen\Fields\Group;
+use Orchid\Screen\Fields\Input;
+use Orchid\Screen\Fields\Relation;
+use Orchid\Screen\Layouts\Rows;
+
+class AddSiteLayout extends Rows
```
Comment (jg-rivera) [inline app/Orchid/Layouts/Batch/AddSiteLayout.php:14]: Should be called `BatchEditSiteLayout`

File: app/Orchid/Layouts/Batch/AddFarmersLayout.php [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+namespace App\Orchid\Layouts\Batch;
+
+use App\Models\Batch\Farmers;
+use App\Models\User;
+use Orchid\Screen\Field;
+use Orchid\Screen\Layouts\Rows;
+use Orchid\Screen\Fields\Relation;
+
+class AddFarmersLayout extends Rows
```
Comment (jg-rivera) [inline app/Orchid/Layouts/Batch/AddFarmersLayout.php:11]: Should be called `BatchEditFarmersLayout`

File: resources/views/livewire/create-batch.blade.php [hunk 5]
```diff
@@ -0,0 +1,37 @@
+
```
Comment (jg-rivera) [inline resources/views/livewire/create-batch.blade.php:1]: Delete unused `create-batch.blade.php`

**Review by jg-rivera — CHANGES_REQUESTED** [review]

File: app/Http/Livewire/CreateBatch.php (hunk 1 above)
Comment (AldwinVergara) [reply app/Http/Livewire/CreateBatch.php:7]: Deleted the unussed livewire component.

File: app/Models/Batch/Batches.php (hunk 2 above)
Comment (AldwinVergara) [reply app/Models/Batch/Batches.php:13]: Changed Batches model to 'Batch'. Also changed it to screen referencing to the model.

File: app/Orchid/Layouts/Batch/AddSiteLayout.php (hunk 3 above)
Comment (AldwinVergara) [reply app/Orchid/Layouts/Batch/AddSiteLayout.php:14]: Renamed 'AddSiteLayout' to 'BatchEditSiteLayout'. Also renamed it on the BatchEditScreen.

File: resources/views/livewire/create-batch.blade.php (hunk 5 above)
Comment (AldwinVergara) [reply resources/views/livewire/create-batch.blade.php:1]: Deleted unused blade.

File: app/Orchid/Layouts/Batch/AddFarmersLayout.php (hunk 4 above)
Comment (AldwinVergara) [reply app/Orchid/Layouts/Batch/AddFarmersLayout.php]: Renamed 'AddFarmersLayout' to 'BatchEditFarmersLayout'. Also renamed it on the BatchEditScreen.

**Review by jg-rivera — APPROVED** [review]

**Pull request merged**